client/signup/steps/woocommerce-install/confirm/index.tsx

```diff
@@ -167,12 +153,11 @@ export default function Confirm( {
 			allowBackFirstStep={ true }
 			backUrl="/woocommerce-installation"
 			headerText={ headerTitle }
-			fallbackHeaderText={ headerTitle }
```

Comment:
removing this prop triggers the following error in the client

![image](https://user-images.githubusercontent.com/77539/143261063-2d3644bf-7bf5-43a8-b2f8-8d2291bb66ae.png)

Reply:
Why is the fallback header being sent in the prop headerText when we sent headerText?